tests/test_workflow.py

```diff
@@ -267,8 +267,8 @@ def test_construct_parameters_parallel(workflow_mock):
 def test_construct_parameters_parallel_multiple_intervals(workflow_mock):
     parameters_list = workflow_mock.construct_parameters_parallel(
         geometries=[
-            shapely.geometry.point.Point(1, 3),
-            shapely.geometry.point.Point(1, 5),
+            box(0.99999, 2.99999, 1.00001, 3.00001),
+            box(0.99999, 4.99999, 1.00001, 5.00001),
```

Comment:
:) nah, not at all. Just copied from different example. Since this just checks if the parameters are created successfully will keep it now, but yeah could look nicer.